### DISCUSSION TITLE
Convert PDF to Black and White

### DISCUSSION THREAD

**Discussion by Gaxnfass** [ImageMagick/ImageMagick#2110] [Help]
I know there have been a lot of threats already concerning this topic, but it didn´t work for me.
I want every black or dark grey pixel to be black and everything else to be white, in order to clean up the blurry PDF.
Thanks in advance for your help.

**Comment by fmw42** [comment]
I assume you mean "threads" and not "threats".

What is your ImageMagick version and platform/OS?  What was your exact command line?  Can you provide an example PDF that does not work as you expected?

**Reply by Gaxnfass** [reply]
Version: ImageMagick 7.0.10-16 OS: MacOS Catalina
Well I tried magick convert original.pdf -threshold 50% output.pdf, because I saw that on another forum, but I don´t know at all what I´m doing.
But the result was just a blank output file. All the pages were there, but blank

**Reply by fmw42** [reply]
Can you post the PDF you used?

**Reply by Gaxnfass** [reply]
Maybe a page of it, but its too big.
[bw.pdf](https://github.com/ImageMagick/ImageMagick/files/4737854/bw.pdf)

**Reply by fmw42** [reply]
Try the following. It works for me on IM 7.0.10.16 Q16 Mac OSX Sierra.

`magick -density 300 bw.pdf -alpha off -colorspace gray -threshold 50% new_bw.pdf`

Note that for IM 7, use magick, not magick convert.  The other tools do need prefacing by "magick", such as magick identify or magick montage, etc.

**Reply by Gaxnfass** [reply]
Thanks a lot.
